Dockerfile

```diff
@@ -1,22 +1,9 @@
-FROM golang
+FROM multichain-base:latest
```

Comment:
Is it possible to call this `multichain`?

Reply:
@jazg I'll change this to `multichain:latest`, but this means you will need to build the `multichain` image as well. I have not pushed the `multichain` image to docker hub.

Reply:
@roynalnaruto Can we push the `multichain` image to Docker Hub?

Reply:
Yes, that's the next step, automated build and push of `renproject/multichain` to Docker hub. The lightnode dockerfile would then use:
```
FROM renproject/multichain:latest
```